plugins/chain_api_plugin/chain_api_plugin.cpp

```diff
@@ -50,7 +50,7 @@ parse_params<chain_apis::read_only::get_transaction_status_params, http_params_t
 
 #define CALL_WITH_400(api_name, api_handle, api_namespace, call_name, http_response_code, params_type) \
 {std::string("/v1/" #api_name "/" #call_name), \
-   [api_handle](string, string body, url_response_callback cb) mutable { \
+   [api_handle](string&&, string&& body, url_response_callback&& cb) mutable { \
```

Comment:
and similar change in this file in line 157?
`( string&&, string&& body, url_response_callback&& cb ) mutable {`

Reply:
Good catch, thanks. While looking at it I also noticed that we were copying (not moving) abi_cache, so I fixed that as well.